src/Uno.UI/UI/Xaml/Input/PointerRoutedEventArgs.wasm.cs

```diff
@@ -15,7 +16,7 @@
 
 namespace Windows.UI.Xaml.Input
 {
-	partial class PointerRoutedEventArgs
+	partial class PointerRoutedEventArgs : IHtmlHandleableRoutedEventArgs
```

Comment:
Maybe:
```suggestion
	partial class PointerRoutedEventArgs : ICanHandleHtmlRoutedEventArgs
```

Reply:
This is to match the already existing [`IHandleableRoutedEventArgs`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Input/Internal/IHandleableRoutedEventArgs.cs)